perinatal-hiv/0-2-copyright.md

```diff
@@ -24,4 +24,4 @@ This licence means you may share, copy and redistribute the material in any medi
 * Non-Commercial — You may not use the material for commercial purposes.
 * No Derivatives — If you remix, transform, or build upon the material, you may not distribute the modified material.
 
-For any other uses, please contact [[Bettercare](http://bettercare.co.za).
+For any other uses, please contact [[Bettercare](http://bettercare.co.za).
```

Comment:
@nkoeries You're going to get a square bracket showing here.

Reply:
@arthurattwell fixed.